main.go

```diff
@@ -109,11 +104,83 @@ func main() {
 	config.BreakingOnly = breakingOnly
 	config.DeprecationDays = deprecationDays
 
-	diffReport, err := diff.Get(config, s1, s2)
+	var diffReport *diff.Diff
+	var err error
+	var operationsSources *diff.OperationsSourcesMap
 
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "diff failed with %v\n", err)
-		os.Exit(104)
+	if checkBreaking {
+		config.IncludeExtensions.Add(checker.XStabilityLevelExtension)
+		config.IncludeExtensions.Add(diff.SunsetExtension)
+		config.IncludeExtensions.Add(checker.XExtensibleEnumExtension)
+	}
+
+	if composed {
+		s1, err := load.FromGlob(loader, base)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to load base spec from %q with %v\n", base, err)
+			os.Exit(102)
+		}
+
+		s2, err := load.FromGlob(loader, revision)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to load revision spec from %q with %v\n", revision, err)
+			os.Exit(103)
+		}
+		diffReport, operationsSources, err = diff.GetPathsDiff(config, s1, s2)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "diff failed with %v\n", err)
+			os.Exit(104)
+		}
+	} else {
+		s1, err := checker.LoadOpenAPISpecInfoFromFile(base)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to load base spec from %q with %v\n", base, err)
+			os.Exit(102)
+		}
+
+		s2, err := checker.LoadOpenAPISpecInfoFromFile(revision)
```

Comment:
This change prevents loading files from URLs which was previously supported, can you please check?

Reply:
Thanks, fixed